testbed/coav-sim.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
+
+# Supported Autopilots
+AP_PX4=0
+AP_APM=1
+
+# ArduCopter Variables
+APM_CMD="arducopter"
+APM_TCP_PORT_1=5760
+APM_TCP_PORT_2=5762
+
+# PX4 Variables
+PX4_DIR=${PX4_DIR:-"~/px4/Firmware"}
+PX4_CMD="make posix_sitl_default jmavsim"
+PX4_UDP_PORT_1=14550
+PX4_UDP_PORT_2=14540
+
+# Collision Avoidance and Gazebo variables
+GZSITL_UDP_PORT=15556
+COAV_GCS_UDP_PORT=15557
+
+# Set autopilot
+AUTOPILOT=${AUTOPILOT:-AP_PX4}
+
+run_autopilot () {
+        # Run SITL Simulator
+    if (("$AUTOPILOT" == "$AP_PX4")); then
+        cd $PX4_DIR
+        $PX4_CMD &
+        SITLID=$!
+        cd - > /dev/null
+
+        # Wait for sitl
+        sleep 15
+    elif (("$AUTOPILOT" == "$AP_APM")); then
+        cd $SCRIPT_DIR # sitl must run in the same dir of "eeprom.bin"
```

Comment:
Actually it does. That eeprom has the drone parameters pre loaded. Should it really need it? I don't think so. I think we can find a way to get rid of it. Right now running without loading that file prevents the drone from taking off sometimes. But anyway, I'll look into removing this.